packages/ui/components/document/document-share-button.tsx

```diff
@@ -71,7 +71,6 @@ export const DocumentShareButton = ({ token, documentId, className }: DocumentSh
 
   const onTweetClick = async () => {
     let { slug = '' } = shareLink || {};
-
     if (!slug) {
       const result = await createOrGetShareLink({
         token,
```

Comment:
Apologies if it wasn't clear, we can make the `token` prop that this component accepts optional. If you look at the `createOrGetShareLink` implementation you'll find it's optional there too defaulting to the authenticated user if available.

This means that we don't need to using nullish coalescing in the `data-table-action-dropdown` component 😄

Reply:
sure , just made the token props as optional .